astroplan/core.py

```diff
@@ -269,57 +551,98 @@ def moon_position(self, time):
         """
         raise NotImplementedError()
 
-    # Other time-related methods.
+    # Twilight convenience functions
 
-    def evening_nautical(self, time):
+    def evening_astronomical(self, time, which='nearest'):
         """
-        Returns the local evening (nautical) twilight time.
+        Time at evening astronomical (-18 degree) twilight.
 
         Parameters
         ----------
         time : `~astropy.time.Time`
+            Time of observations
+
+        which : str - can be "next", "previous", or "nearest"; defaults to "nearest"
+            Choose which twilight relative to the present `time` would you
+            like to calculate
         """
-        raise NotImplementedError()
+        return self.sunset(time, which, horizon=-18*u.degree)
+
+    def evening_nautical(self, time, which='nearest'):
 
-    def evening_civil(self, time):
         """
-        Returns the local evening (civil) twilight time.
+        Time at evening nautical (-12 degree) twilight.
 
         Parameters
         ----------
         time : `~astropy.time.Time`
+            Time of observations
+
+        which : str - can be "next", "previous", or "nearest"; defaults to "nearest"
+            Choose which twilight relative to the present `time` would you
+            like to calculate
         """
-        raise NotImplementedError()
+        return self.sunset(time, which, horizon=-12*u.degree)
 
-    def morning_astronomical(self, time):
+    def evening_civil(self, time, which='nearest'):
         """
-        Returns the local morning (astronomical) twilight time.
+        Time at evening civil (-6 degree) twilight.
 
         Parameters
         ----------
         time : `~astropy.time.Time`
+            Time of observations
+
+        which : str - can be "next", "previous", or "nearest"; defaults to "nearest"
+            Choose which twilight relative to the present `time` would you
+            like to calculate
         """
-        raise NotImplementedError()
+        return self.sunset(time, which, horizon=-6*u.degree)
 
-    def morning_nautical(self, time):
+    def morning_astronomical(self, time, which='nearest'):
         """
-        Returns the local morning (nautical) twilight time.
+        Time at morning astronomical (-18 degree) twilight.
 
         Parameters
         ----------
         time : `~astropy.time.Time`
+            Time of observations
+
+         which : str - can be "next", "previous", or "nearest"; defaults to "nearest"
```

Comment:
This isn't formatted correctly in the HTML output. It's not obvious to me what's wrong ... maybe the semicolon.